### PR TITLE
Improve CLI docs

### PR DESCRIPTION
**Pull request by dstansby**
- Use bash highlighting
- Break up code blocks so it's easier to copy/paste commands
- Remove leading `$` so it's easier to copy/paste commands

### CODE REVIEW

**Review by will-moore — APPROVED** [review]
Looks good at https://ome-zarr--367.org.readthedocs.build/en/367/cli.html 
Thanks

File: docs/source/cli.rst [hunk 1]
```diff
@@ -7,50 +9,53 @@ The examples below use the image at http://idr.openmicroscopy.org/webclient/?sho
...
-    # Local data (after downloading as below)
-    $ ome_zarr view 6001240.zarr/
+    ome_zarr view 6001240.zarr/
 
 download
 ========
 
 To download all the resolutions and metadata for an image::
 
-    # creates local 6001240.zarr/
```
Comment (will-moore) [inline docs/source/cli.rst:37]: I think this line is still useful to keep, maybe add to the sentence above?

Comment (dstansby) [reply docs/source/cli.rst:37]: 👍 done

**Review by joshmoore — CHANGES_REQUESTED** [review]
Sphinx is failing with:

```
/home/docs/checkouts/readthedocs.org/user_builds/ome-zarr/checkouts/367/docs/source/cli.rst:38: WARNING: Inline literal start-string without end-string.
```
see https://beta.readthedocs.org/projects/ome-zarr/builds/24167312/

**Review by will-moore — APPROVED** [review]
LGTM: thanks!

**Comment by joshmoore** [comment]
Codecov took months to complete?

**Pull request merged**